server/server.go

```diff
@@ -514,11 +514,14 @@ func (s *Server) onConn(conn *clientConn) {
 			})
 			terror.Log(err)
 		}
-		if errors.Cause(err) == io.EOF {
+		switch errors.Cause(err) {
+		case io.EOF:
 			// `EOF` means the connection is closed normally, we do not treat it as a noticeable error and log it in 'DEBUG' level.
 			logutil.BgLogger().With(zap.Uint64("conn", conn.connectionID)).
 				Debug("EOF", zap.String("remote addr", conn.bufReadConn.RemoteAddr().String()))
-		} else {
+		case errConCount:
+			_ = conn.writeError(ctx, err)
```

Comment:
```suggestion
			_ = conn.writeError(ctx, err)
```
Please handle the error here, you could simply write an log for it.

Reply:
Fixed. PTAL :)